govcd/sample_govcd_test_config.json

```diff
@@ -21,7 +22,7 @@
 			"name": "mycat",
 			"//": "One item in the catalog. It will be used to compose test vApps. Some tests",
 			"//": "rely on it being Photon OS. It is is not Photon OS - the tests will be skipped",
-			
+
```

Comment:
Not sure if there is a reason for this empty line.

Reply:
fixed